modules/metastorage/README.md

```diff
@@ -0,0 +1,40 @@
+# Metastorage
+
+The module for store and access to metadata.
+
+The storage is distributed to prevent losing data and based of RAFT. In typical case, the group consists of only several nodes of the
```

Comment:
Failure tolerance has the same stance in case of one node leaves the cluster and data should be still available. In my mind, the prevention of losing data is a general characteristic for all requirements that an any distributed system has.